services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/keyfigurecard/factory/ReproductionNumberCardFactory.java

```diff
@@ -38,8 +38,8 @@ private KeyFigure getSevenDayRValueKeyFigure(StatisticsJsonStringObject stats) {
   @Override
   protected List<Optional<Object>> getRequiredFieldValues(StatisticsJsonStringObject stats) {
     return List.of(
-        Optional.ofNullable(stats.getSevenDayRvalue1stReportedTrend1percent()),
-        Optional.ofNullable(stats.getSevenDayRvalue1stReportedDaily())
+        Optional.ofNullable(stats.getSevenDayRvaluePublishedTrend1percent()),
+        Optional.ofNullable(stats.getSevenDayRvaluePublishedDaily())
```

Comment:
@fredrb / @ioangut - what branch is correct? main or release/1.11?

Reply:
release/1.11, as there is the last hotfix